deployment/ansible/roles/cac-tripplanner.otp-data/files/validate_feed.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/python
+
+# Check GTFS feeds have no errors and are current, using feedvalidator.py
```

Comment:
Did you write this script or did it come from somewhere? There are quite a things that don't match our style guidelines, like short var names, no docstrings. Either way, I think it's fine as is, but if it came from somewhere would probably be helpful to cite the source.